### PR TITLE
Security over ear headsets

### PR DESCRIPTION
**Pull request by JoeHammad1844**
## About the PR

Security now start with over ear headsets instead of the in-ear ones

**Media**

image
image
image
image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: JoeHammad
- add: Security now wear over-ear headsets, drippy!

### CODE REVIEW

File: Resources/Prototypes/Roles/Jobs/Security/warden.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@
     jumpsuit: ClothingUniformJumpsuitWarden
     back: ClothingBackpackSecurityFilled
     shoes: ClothingShoesBootsCombatFilled
-    eyes: ClothingEyesGlassesSecurity
+    eyes: ClothingHeadsetAltSecurityOfficer
```
Comment (Ruichka) [inline Resources/Prototypes/Roles/Jobs/Security/warden.yml]: Eyes?

**Comment by JoeHammad1844** [comment]
oopsie poopsie!

**Comment by dmnct** [comment]
Okay, and... why?

**Comment by JoeHammad1844** [comment]
played ss13, they use over ear's, realized it looks cooler

**Comment by JoeHammad1844** [comment]
also, mimimi, I go to sleep now, it is 2AM

**Comment by JoeHammad1844** [comment]
I get up!

**Comment by LankLTE** [comment]
Does this completely remove normal headsets (from being typically obtainable)? That feels odd, keep them in the secdrobe so people have a choice.

**Comment by JoeHammad1844** [comment]
lawyers will still get regular sec headsets

**Comment by LankLTE** [comment]
> lawyers will still get regular sec headsets

That doesn’t really mean they’re obtainable unless you beg a lawyer for their headset. Some people might like the old headsets better so why not put some in the SecDrobe and let them choose

**Comment by metalgearsloth** [comment]
Can't we just wait for loadouts instead

**Comment by JoeHammad1844** [comment]
mimeteen eighty four

**Pull request closed without merging**